### PR TITLE
VACMS-19450: Creates Banner Module

### PR DESCRIPTION
**Pull request by dsasser**
## Summary
Adds a new Banner module to the codebase. This will house the new banner api we are building.

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/19450

## Testing done
Automated using RSPEC tests.

## Screenshots
Screenshot 2024-10-24 at 7 32 17 AM

## What areas of the site does it impact?
Banners

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: modules/banners/README.rdoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+= Banners
+The Banners API provides a means to generate VA.gov banners quickly, without relying on content-build.
+
+== Installation
+Ensure the following line is in the root project's Gemfile:
+
+  gem 'banners', path: 'modules/banners'
```
Comment (SnowboardTechie) [inline modules/banners/README.rdoc:7]: Do we need the path portion here? I noticed it was not necessary in the root Gemfile

**Review by SnowboardTechie — APPROVED** [review]
LGTM, tested locally in console

File: modules/banners/README.rdoc (hunk 1 above)
Comment (dsasser) [reply modules/banners/README.rdoc]: Hmm, probably not, removed.

**Comment by dsasser** [comment]
Closed because some GHA's were not completing successfully that were resolved, so they appeared to be stuck in the failed state.

**Pull request closed without merging**